server/controllers/index.js

```diff
@@ -0,0 +1,169 @@
+var { pool } = require('../db');
+
+const getReviews = (req, res) => {
+  console.log('Getting Reviews', req.query);
+  let reviews = {};
+  reviews.product = req.query.product_id || 1;
+  reviews.page = Number(req.query.page) || 1;
+  reviews.count = Number(req.query.count) || 5;
+  reviews.sort = req.query.sort || 'date';
+  reviews.results = [];
+
+  const limit = reviews.page * reviews.count;
+  if (reviews.sort === 'newest' || reviews.sort === 'relevant') {
+    reviews.sort = 'date';
+  }
+  const query = {
+    text: `SELECT review_id, rating, summary, recommend, response, body, date, reviewer_name, helpfulness FROM reviews WHERE product_id=$1 ORDER BY ${reviews.sort} DESC, date DESC LIMIT $2;`,
+    values: [reviews.product, limit],
+  }
+  pool.query(query)
+    .then((result) => {
+      reviews.results = result.rows;
+      let photosPromise = [];
+      reviews.results.forEach((review) => {
+        const query = {
+          text: `SELECT id, url FROM photos WHERE review_id=$1;`,
```

Comment:
per airbnb guide, "template literals should contain interpolation or newlines", so this should be wrapped in single quotes.